### PR TITLE
starting red flag list

### PR DESCRIPTION
**Pull request by alex-min**
This list is a work in progress. Please comment add things you would consider as red flags.

### CODE REVIEW

File: process/development_practices.md [hunk 1]
```diff
@@ -23,4 +23,13 @@ When a new developer joins our team, we'll often find they need some guidance on
...
+* Client showcases are really important. You should always try to attend these.
+
+
+## Red flags on projects
+
+This is a list of red flags to review every day to improve project delivery:
+
+- [ ] There is no showcase currently planned for the current piece of work
+- [ ] The current epic is longer than a week (it should be broken down in multiple smaller ones)
+- [ ] Customer does not know when the current epic is expected to be finished
```
Comment (RoryMacDonald) [inline process/development_practices.md]: * Huge commits
* Production deploy > 1 day ago
* Not being clear on who the customer is

Comment (alex-min) [inline process/development_practices.md]: :+1:

**Comment by craigjbass** [comment]
Nice

File: process/development_practices.md (hunk 1 above)
Comment (SebAshton) [inline process/development_practices.md]: Insufficient information to plan iteration. e.g. missing designs

Comment (SebAshton) [inline process/development_practices.md]: @nickwood might have his red flag list stashed somewhere too...

Comment (alex-min) [inline process/development_practices.md]: Thanks Seb, I'll also add that to the list, this is in a WIP state!

**Pull request merged**